ADOPTERS.md

```diff
@@ -0,0 +1,8 @@
+Kroxylicious adopters
```

Comment:
Isn't the file name more conventionally `ADOPTERS.md`

Reply:
You have me slightly confused with this one, isn't that what I called the file? I fully expect I could have typo'd it but your suggestion is the same as what I have.

Reply:
I did get it wrong in the branch name, but the file in the PR is right.